skimage/measure/_regionprops_utils.py

```diff
@@ -183,16 +184,19 @@ def euler_number(image, connectivity=None):
         return int(0.125 * coefs @ h)
 
 
-def perimeter(image, neighbourhood=4):
+@deprecate_kwarg(kwarg_mapping={'neighbourhood': 'neighborhood'},
+                 removed_version="1.0",
+                 deprecated_version="0.19.1")
```

Comment:
```suggestion
                 deprecated_version="0.19.2")
```

This was not yet deprecated in 0.19.1. I don't think we usually add deprecations in patch releases, either, though? We could just deprecate in the next release, but then removed version would be something like 1.2. I don't think we made a final decision on "1.0" vs. "2.0" though? (another reason to have a team meeting soon!)